src/main/cpp/t09_spiral.cpp

```diff
@@ -17,11 +17,48 @@
 
 #include "t09_spiral.h"
 #include <iostream>
+#include <iomanip>
 
 
 using namespace std;
 
 int t09_spiral() {
-    
-    return 0;
+        int n, m, i, j, b = 0;
+    cin >> n >> m;
+    int a[n][m];
+
+    for (i = 0; i < n; i++){
+        for (j = 0; j < m; j++){
+            a[i][j] = 0;
+        }
+    }
+    i = 0, j = -1; 
+    while (b < n * m) {
+        while (j < m - 1 & a[i][j+1]==0){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Boolean result is used in bitwise operation. Clarify expression with parentheses.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4729584)